flask_admin/contrib/sqla/view.py

```diff
@@ -853,27 +854,37 @@ def _apply_filters(self, query, count_query, joins, count_joins, filters):
         for idx, flt_name, value in filters:
             flt = self._filters[idx]
 
-            alias = None
-            count_alias = None
+            aliases = None
+            count_aliases = None
 
             # Figure out joins
             if isinstance(flt, sqla_filters.BaseSQLAFilter):
-                path = self._filter_joins.get(flt.column, [])
-
-                query, joins, alias = self._apply_path_joins(query, joins, path, inner_join=False)
-
-                if count_query is not None:
-                    count_query, count_joins, count_alias = self._apply_path_joins(
-                        count_query,
-                        count_joins,
-                        path,
-                        inner_join=False)
+                aliases = []
+                count_aliases = []
+                is_multiple_columns = isinstance(flt.column, collections.Sequence) and not isinstance(flt.column, str)
```

Comment:
Probably need to use `text_type` here instead of `str` for python 3 compatibility? text_type comes from `from flask_admin._compat import text_type`